client/index.html

```diff
@@ -343,6 +343,13 @@ <h1>Settings</h1>
                             </template>
                         </micro-options>
                     </label>
+
+                    <label>
+                        Item template
```

Comment:
The margin seems of here - would be awesome if you could correct this: In *micro.css*, there is a rule for `form > label`, I think it is sufficient to just change it to `label` (maybe have a quick look around if other forms are broken then ;) ).